tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+find_program(XEMU NAMES xmega65 xmega65.native)
+set(XEMU_ARGS "-besure -headless -sleepless -testing")
+
+function(TEST NAME)
+    add_executable(${NAME} ${NAME}.c)
+    target_link_libraries(${NAME} mega65libc)
+    set_target_properties(${NAME} PROPERTIES OUTPUT_NAME ${NAME}.prg)
+    if (XEMU)
+        add_test(NAME ${NAME} COMMAND sh -c "${XEMU} ${XEMU_ARGS} -prg ${NAME}.prg")
+    endif()
+endfunction()
+
+# compile example prg file
+add_executable(time-test time.c)
```

Comment:
what is the purpose of the time.prg executable?

Reply:
None - will remove from #24.